appveyor.yml

```diff
@@ -0,0 +1,34 @@
+version: 1.0.{build}-{branch}
+
+environment:
+  matrix:
+    - RUBY_VERSION: 25
+    - RUBY_VERSION: 25
+      CAPYBARA_FF: true
+    - RUBY_VERSION: 24
+    - RUBY_VERSION: 24
+      CAPYBARA_FF: true
+    - RUBY_VERSION: 23
+    - RUBY_VERSION: 23
+      CAPYBARA_FF: true
+    - RUBY_VERSION: 22
+    - RUBY_VERSION: 22
+      CAPYBARA_FF: true
+
+install:
+  - set PATH=C:\Ruby%RUBY_VERSION%\bin;%PATH%
+  - cinst selenium-gecko-driver selenium-chrome-driver Firefox GoogleChrome
+  - bundle config --local path vendor/bundle
+  - bundle install
+
+build: off
+
+before_test:
+  - ruby -v
+  - gem -v
+  - bundle -v
+  - chromedriver --version
```

Comment:
logging of driver version in the browser specific specs can be made dependent on ENV['CI'] - then this can go away